src/Microsoft.Health.Fhir.Api.UnitTests/Controllers/ExportControllerTests.cs

```diff
@@ -25,17 +27,11 @@ public ExportControllerTests()
         }
 
         [Fact]
-        public void GivenAnExportRequest_WhenDisabled_ThenRequestNotValidExceptionShouldBeThrown()
+        public async void GivenAnExportRequest_WhenDisabled_ThenRequestNotValidExceptionShouldBeThrown()
```

Comment:
should this be `async Task`?

Reply:
Since I am just asserting that an exception is being thrown and not returning anything, I went with `async void` instead of `async Task`.